sdk/keyvault/Azure.Security.KeyVault.Keys/tests/AesCryptographyProviderTests.cs

```diff
@@ -138,6 +138,7 @@ public void DecryptionAlgorithmNotSupported()
         }
 
         [TestCaseSource(nameof(GetEncryptionAlgorithms), methodParams: new object[] { true })]
+        [IgnoreOnNet5("https://github.com/Azure/azure-sdk-for-net/issues/16968")]
```

Comment:
I see you did include a link. I'd just call "reason" "issueLink" instead to reinforce that.